aten/src/ATen/native/Embedding.cpp

```diff
@@ -144,19 +144,17 @@ Tensor & embedding_renorm_cpu_(
   auto sorted_indices = std::vector<int64_t>(data_ptr, data_ptr + num_indices);
   std::sort(sorted_indices.begin(), sorted_indices.end(), std::less<int64_t>());
 
-  at::parallel_for(0, num_indices, 1000, [&](int64_t start, int64_t end) {
-    for (auto i = start; i < end; i++) {
-      if (i > 0 && sorted_indices[i] == sorted_indices[i - 1]) {
-        continue;
-      }
-      auto row = self[sorted_indices[i]];
-      auto norm = row.norm(norm_type).item<double>();
-      if (norm > max_norm) {
-        auto scale = max_norm / (norm + 1e-7);
-        row *= scale;
-      }
+  for (auto i = 0; i < num_indices; i++) {
```

Comment:
you should comment this -- it would be totally sensible for someone to not realize what's going on here and change it back.